modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -195,14 +195,14 @@ pub struct PaymentFee {
 impl PaymentFee {
     pub fn one_percent() -> Self {
         PaymentFee {
-            base: Amount::ZERO,
+            base: Amount::from_sats(50),
             parts_per_million: 10_000,
         }
     }
 
     pub fn half_of_one_percent() -> Self {
         PaymentFee {
-            base: Amount::ZERO,
```

Comment:
I think we should change the names of the above functions since they are no longer percentages

Reply:
Sure

Reply:
Done